Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 # Jumbo Jekyll Theme 
-gem 'jumbo-jekyll-theme', "3.9.4"
+gem 'jumbo-jekyll-theme', "5.5.5"
```

Comment:
PR for the `jekyll-build-container` (Dockerfile)[https://github.com/linaro-its/jekyll-build-container/blob/master/Dockerfile] please. :smile: Please also update the `Used by` lines as appropriate.

Reply:
Platforms doesn't use the Docker build process yet, @morancj